src/scripts/sync_synonym.py

```diff
@@ -86,6 +121,23 @@ def _filter_format_save(
     return df
 
 
+# todo: consider refactoring all .drop_duplicates() to here
+def _read_sparql_output_tsv(path: Union[Path, str]) -> pd.DataFrame:
+    """Read and format special kind of TSV that comes from SPARQL query output."""
+    return pd.read_csv(path, sep='\t').fillna('').rename(columns=lambda x: x.lstrip('?'))
+
+
+def _remove_language_code(text: str, pattern=r'@([a-zA-Z]{2,3}(?:-[a-zA-Z-]+)?)$') -> str:
+    """Remove language code from the end of string.
```

Comment:
## Language code removal
- [ ] @twhetzel @matentzn Review

### What this is doing
> `pattern`: Default will capture either of these cases, 
> (1) String ends with @ followed by 2-3 characters,
> (2) String ends with @ followed by 2-3 characters, followed by a -, followed by an number of characters.

> Complies with RDF & OWL's language tag formatting based on IETF's BCP 47 (Best Current Practice 47) standard.

### Why a change like this was necessary
I couldn't use OAK to query synonym types, so I used SPARQL. But unlike with OAK (or SemSQL), many synonyms are coming through with `@en`, some `@fr`, and possibly others.  I think 3-4 sources had these (DO, Orphanet, and ICD10CM at least I think).

Using Claude I was easily able to make a regex meeting those conditions above, which handles simple cases, e.g. `@fr`, but also more complex cases like `@zh-Hans-CN`. I did some quick true/false negative/positive testing, and it passed my checks.

### What I'd like to do in the future
Find an existing library for this, or otherwise implement this in `mondolib`.

### What I'd like from review
Just wanna know if you guys feel dubious about the regex. If so, then maybe for now we can go with a simple check for `@en` and `@fr`. However, I don't know if this will 100% cover all cases for our current sources without further analysis.

Reply:
This (removing any language tags like `@en`) should be addressed in the SPARQL query.

Reply:
I was thinking of that but when this has come up before we've done it in Python. I'll look into how to do that in the SPARQL query.

I think probably this change will need go into next week's changs.